x-pack/plugins/index_management/server/routes/api/indices/register_get_route.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { IScopedClusterClient } from '@kbn/core/server';
+import { schema } from '@kbn/config-schema';
+import { KbnServerError, reportServerError } from '@kbn/kibana-utils-plugin/server';
+import { fetchIndices } from '../../../lib/fetch_indices';
+import { RouteDependencies } from '../../../types';
+import { addInternalBasePath } from '..';
+
+export function registerGetRoute({
```

Comment:
This route is not hidden behind the feature flag, since I believe this is not affecting end users. But let me know what you think.